_posts/2024-08-20-release-0_7_0.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Kroxylicious release 0.7.0"
+date:   2024-08-20 00:00:00 +0000
+author: "Keith Wall"
+author_url: "https://www.github.com/k-wall"
+categories:  [releases, kroxylicious, record-validation]
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.7.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.7.0). See the [Changelog](https://github.com/kroxylicious/kroxylicious/blob/main/CHANGELOG.md#070) for a list of changes and summary of Deprecations, Changes and Removals.
+
+### Record Validation 
+
+In this release, Kroxylcious dips its toes (or, should that be claws?) into the world of record validation.  You can now ensure that the content your producers send adheres to the schema you expect.   If a producer sends
+a message that doesn't meet the expected schema, the producer will get an error message and the message won't be delivered to the topic. Banish those poison messages from your topics for good!
+Read more about the [use-case](https://kroxylicious.io/use-cases/#schema-validation-and-enforcement) or dive right into the [documentation](https://kroxylicious.io/docs/v0.7.0/#assembly-record-validation-filter-proxy).
+This is a preview feature and we are already thinking about enhancements. We'd love to hear your feedback. Thanks to [Carles Arnal](https://github.com/carlesarnal) for all the work he put into this feature.
```

Comment:
Wondering if we should link to ApiCurio somewhere in here

Reply:
I couldn't quite find the right formulation of words to include Apicurio in the first couple of sentences whilst keeping things snappy.   Feel free to reword the announcement.   I cheated and added an apicurio label and a mention of Carles's project affiliation.